### ISSUE TITLE
Training Details of FedAvg

### ISSUE THREAD

**Issue by wnn2000** [Xu-Jingyi/FedCorr#4]
Hi Jingyi and Zihan,

Thanks for your code. I’m having difficulty achieving the performance of FedAvg on clean CIFAR-10 that you mentioned in Table 2 (acc: 93.11±0.12). Could you please provide some details? For instance, the number of online clients, the optimizer used, and whether Mixup was applied?

Best, 
Nannan

**Comment by Xu-Jingyi** [comment]
Hi Nannan,

Thank you for taking an interest in our work.

We used the following command for FedAvg. You can try to see if you can achieve the same accuracy.
python baseline.py --beta 0 --momentum 0.5 --num_users 100 --dataset cifar10 --rounds 500 --frac 0.1 --model resnet18 --lr 0.03 --local_ep 5 --level_n_system 0 --level_n_lowerb 0 --seed 1

Best,
Jingyi

**Comment by wnn2000** [comment]
Hi Jingyi,

Thanks for your reply. I have found that using your resnet18 implementation can lead to an accuracy similar to that in your paper. However, using resnet18 provided by `torchvision` seems to lead to a 10% drop in accuracy. Do you know the reason?

Best,
Nannan

**Comment by Xu-Jingyi** [comment]
Hi Nannan,

Sorry, I don't know the reason, the architecture should be the same. Maybe you can try a different learning rate for the resent18 provided by torchvision.

Best,
Jingyi
